spec/active_record/monetizable_spec.rb

```diff
@@ -121,18 +132,16 @@ class SubProduct < Product
       end
 
       context "when MoneyRails.raise_error_on_money_parsing is true" do
-        before { MoneyRails.raise_error_on_money_parsing = true }
+        before do
+          MoneyRails.raise_error_on_money_parsing = true
+          Money.default_bank.add_rate("RUB", "USD", 1)
+        end
+
         after { MoneyRails.raise_error_on_money_parsing = false }
 
         it "raises exception when a String value with hyphen is assigned" do
           expect { product.accessor_price = "10-235" }.to raise_error ArgumentError
         end
-
-        it "raises an exception if it can't change currency" do
-          expect {
-            Product.new.price = Money.new(10, "RUB")
-          }.to raise_error(MoneyRails::ActiveRecord::Monetizable::ReadOnlyCurrencyException, "Can't change readonly currency 'USD' to 'RUB' for field 'price'")
-        end
```

Comment:
Because beforehand exchange_to is called. By doing that Money::Bank::UnknownRate will be raised instead of ReadOnlyCurrencyException.
I think this part is covered elsewhere (in the money gem?). But if you want I can rewrite this part as well.